src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,17 @@
+body,
+div,
+p,
+a {
+  margin: 0;
+  font-family: $font-family;
+}
```

Comment:
1. You forgot to reset margins for `<h2>`
2. `<a>`, `<div>` doesn't have margins
3. It is better to define `font-family: $font-family;` in `html` or `:root`